### PR TITLE
Fb cir 1890 adminaccounts

### PR DESCRIPTION
**Pull request by malpania**

### CODE REVIEW

File: src/com/indicee/api/connector/IndiceeAPIConnector.java [hunk 1]
```diff
@@ -909,49 +923,121 @@ public String makePostCallRtnString(String path, String body) throws IndiceeAPIC
...
+     *            The parameters to send to the server. If null no parameters
+     *            will be sent. If this contains a {@link File} or
+     *            {@link InputStreamWrapper} it will be properly sent to
+     *            Indicee.
+     * @return A parsed xml {@link Document} of the server response.
+     * @throws IndiceeAPIConnectorException
+     *             Thrown if there was an error connecting to the server, or if
+     *             the server returned an error.
+     */
+    public Document makePutCallRtnDocument(String path, Map<String, ? extends Object> parameters)
```
Comment (kcpublic) [inline src/com/indicee/api/connector/IndiceeAPIConnector.java]: If not used, please remove.

File: src/com/indicee/api/parser/AccountParser.java [hunk 2]
```diff
@@ -191,4 +191,17 @@ public static Account createAccount(String userDocument) {
         }
         return parameter;
     }
+
+    public static List<Long> getList(Node node) {
```
Comment (kcpublic) [inline src/com/indicee/api/parser/AccountParser.java]: Add unit tests.  If any level of loops returns wrong result, you're toast.

File: src/com/indicee/api/services/impl/AccountServiceImpl.java [hunk 3]
```diff
@@ -280,33 +290,75 @@ public Account updateAccount(Account account) throws IndiceeAPIConnectorExceptio
...
+    }
+
+    @Override
+    public void removeManagedAccounts(long accountId, List<Long> accounts) throws IndiceeAPIConnectorException {
+        Map<String, Object> parameters = new HashMap<String, Object>();
+        parameters.put("accounts", getStringFromListAccount(accounts));
+        connector.makePutCallRtnString(ACCOUNTS_URL + URL_SLASH + accountId + MANAGED_USERS, parameters);
+    }
+
+    private StringBuilder getStringFromListAccount(List<Long> accounts) {
```
Comment (kcpublic) [inline src/com/indicee/api/services/impl/AccountServiceImpl.java]: Make protected, add @VisibleForTesting annotation, and test the various paths.

File: src/com/indicee/api/connector/IndiceeAPIConnector.java (hunk 1 above)
Comment (malpania) [inline src/com/indicee/api/connector/IndiceeAPIConnector.java]: Fixed, Removed unused method

File: src/com/indicee/api/parser/AccountParser.java (hunk 2 above)
Comment (malpania) [inline src/com/indicee/api/parser/AccountParser.java]: Created Test class AccountParserTest.java

File: src/com/indicee/api/services/impl/AccountServiceImpl.java (hunk 3 above)
Comment (malpania) [inline src/com/indicee/api/services/impl/AccountServiceImpl.java]: Moved method to AccountParser.java and added tests.

**Pull request merged**